### PR TITLE
Provide an Node-only development option for Elm development

### PR DESCRIPTION
**Pull request by bholtbholt**
I don't have a Python environment on my computer, and I've never used one so I assumed I'd get stuck somewhere in setup while working on the Elm portion. Since Elm is strongly typed, I was able to understand the contract between the front-end and back-end and create a mock-server to use.

This PR adds `json-server` with a mock database so it's easy to create various states on the front-end while not affecting the back-end code (or even running it). It's not required for the project, but it makes Elm and CSS development much easier 😄 

I filled out the README with a step by step guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,12 +27,95 @@ Django administration:
...
 
 If you're running a Node environment, you can run `npm run elm-make` instead.
 
+### Front-end Only Local Development
+With a Node environment you can run the entire project without Python. This is
+useful for working on Elm or CSS independently and uses
+[json-server](https://github.com/typicode/json-server) to mock the database.
+
+1. run `npm install` in your CLI to install Elm and JSON server
+1. Duplicate `./json-server/db.sample.json` as `./json-server/db.json`
```
Comment (kyoung) [inline README.md]: this is a great readme addition, but could be made better here by something more along the lines of:
1. Duplicate the sample dataset (optional) with `cp ./json-server/db.sample.json ./json-server/db.json`

File: README.md [hunk 2]
```diff
@@ -26,12 +27,95 @@ Django administration:
...
+#### JSON Server: Modes
+There are two distinct modes in Unconf: **Pitching** and **Schedule**. In
+Pitching you'll see all the available pitches and votes, whereas Schedule
+arranges the pitches into a schedule.
+
+The app starts in Pitching mode. To change it to Schedule:
+
+1. Open `./json-server/routes.json`
+1. Change `"/pitches/mode": "/modes/0",` to `"/pitches/mode": "/modes/1",`,
+where `0` maps to the Pitching UUID, and `1` maps to the Schedule UUID.
```
Comment (kyoung) [inline README.md]: are these UUIDs in the data json?

**Review by kyoung — CHANGES_REQUESTED** [review]
This all seems like a lot more work than just installing python3 :P but I'm happy to get it through so I can get on to the restyled PR. My only request is the commented changes to the docs, and perhaps rather than only supplying the one sample data.json, we prepare several states (voting with a mix of voted and not, scheduled/pitching) and have make targets that symlink or copy each to the canonical data set for quick switching between states, eg. `make state-voting` `make state-schedule` etc.

File: README.md (hunk 2 above)
Comment (bholtbholt) [reply README.md]: Yep, they are: https://github.com/kyoung/unconf/pull/27/files#diff-bc5eebc14fee40b23a29e5a3bbdbc4e9R4 and https://github.com/kyoung/unconf/pull/27/files#diff-bc5eebc14fee40b23a29e5a3bbdbc4e9R8.

These UUIDs are setup to be easy to switch, as 0 and 1. I've [configured](https://github.com/kyoung/unconf/pull/27/files#diff-b281664d0b65dbb811f40ba3b179455dR2) the JSON route to "pick" one of the UUIDs on load, which any contributor can change.

As you've mentioned in the review, I'll set this up so it doesn't need to be changed and users can easily restart the server in the correct mode.

**Comment by bholtbholt** [comment]
_Maybe_ a bit of work compared to installing Python. This took roughly an hour to set up and I guessed getting a successful Python environment, then filling the app and managing states would take me more than that, simply because I'm not that familiar with it.

To be clear **this is not required for the style changes**. It was the way I developed it because of the above Python concerns, but the PRs are completely independent.

Requested changes coming shortly!

**Comment by bholtbholt** [comment]
I like the suggestion of `pitching` and `schedule`. That's much easier to switch back and forth without getting into the internals. 😄 

Pushed those changes, and have attached screenshots (old design) of the two modes.

Screen Shot 2019-05-19 at 9 43 20 AM
Screen Shot 2019-05-19 at 9 43 32 AM

**Review by kyoung — APPROVED** [review]

**Pull request merged**